api/src/main/java/org/openmrs/module/commonreports/reports/PatientHistoryReportManager.java

```diff
@@ -101,9 +102,12 @@
 	@Autowired
 	BasePatientDataLibrary basePatientDataLibrary;
```

Comment:
Let's make this `private` while we're here.

Reply:
Ok..